src/main/java/salesforce/entities/Campaign.java

```diff
@@ -0,0 +1,600 @@
+package salesforce.entities;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+public class Campaign {
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Maybe you can put "@JsonInclude(JsonInclude.Include.NON_NULL)" above the class (line 4). In this way, there is no need to put it above all the attributes